awx/ui/src/screens/Job/JobOutput/useJobEvents.js

```diff
@@ -409,10 +423,37 @@ function getNumCollapsedChildren(node) {
 }
 
 function toggleNodeIsCollapsed(state, eventUuid) {
-  return updateNodeByUuid(state, eventUuid, (node) => ({
+  return {
+    ...updateNodeByUuid(state, eventUuid, (node) => ({
+      ...node,
+      isCollapsed: !node.isCollapsed,
+    })),
+    isAllCollapsed: state.isAllCollapsed ? false : state.isAllCollapsed,
```

Comment:
this is equivalent to `isAllCollapsed: false`, so it should probably be simplified to that.